bool read_E_stop
int8 read_gear
int16 read_speed
int32 read_steer